p4-16/spec/P4-16-spec.mdk

```diff
@@ -6591,8 +6594,10 @@ type, or no value at all (specified by a return type of `void`).
 
 The next table lists restrictions on what kinds of calls can be made
 from which places in a P4 program.  Calling a parser, control, or
-table means invoking its `apply()` method.  The row for `extern`
-describes where extern method calls can be made from.
+table means invoking its `apply()` method.  Calling a `value_set`
```

Comment:
I would say that a value set expression evaluates at runtime to a set of values of the specified type. The evaluation reflects the latest updates made by the control-plane.

Reply:
I agree with that statement.  This comment is just before the table on restrictions of where different kinds of stateful objects can be "called" at run-time.  The closest thing that a value_set has to being called at run-time is to be evaluated when the enclosing select expression is evaluated.  I wasn't intending to define the behavior of a value_set here in this section.